MainTel/MainTel-Troubleshooting/MTCS-Troubleshooting/MTDome-Troubleshooting/MTDome-Recover-and-clear-Faults.rst

```diff
@@ -0,0 +1,122 @@
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+.. _MTDome Status: https://summit-lsp.lsst.codes/chronograf/sources/1/dashboards/165?refresh=10s&lower=now%28%29%20-%2015m
+
+################################
+MTDome Recover and clear Faults
+################################
+
+.. _MTDome-Troubleshooting-MTDome-Recover-and-clear-Faults-Overview:
+
+Overview
+========
+ 
+If issues arise with moving the MTDome directly, or if you observe that the *MTDomeTrajectory* following mode is ``ENABLE``, but the Dome fails to follow the telescope, it is likely that the MTDome is having an internal subsystem ``FAULT``` and requires recovery.
+
+Under specific circumstances, the Dome may ignore move commands, such as when the position is identical to the previous *moveAz* command and the *velocity is 0.0 [deg/sec]*. 
+To confirm this, inspect the MTDome logs in LOVE. 
+The CSC will have received the duplicated move command, and a warning message will be displayed in the logs, indicating that it was ignored.
+
+In other instances, the dome may reject move commands, when there's a subsystem ``FAULT``.
+
+.. _MTDome-Troubleshooting-MTDome-Recover-and-clear-Faults-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+- If a ``FAULT`` is indicated in the *Azimuth Control Software Status* (ACS status) window, the Dome has an internal ``FAULT`` condition.
+- Look at the Chronograf dashboard `MTDome Status`_ to verify the system status.
+
+.. Note::
+    
+    The Dome has several subsystems that continue to operate when a specific subsystem goes to ``FAULT``, the CSC itself won't go to ``FAULT``, unless it is an unrecoverable issue.
+    A subsystem going to ``FAULT`` is recoverable.
+
+..
+.. _MTDome-Troubleshooting-MTDome-Recover-and-clear-Faults-Procedure-Steps:
+
+Procedure Steps
+===============
+1. Disable the Dome following mode in *MTDomeTrajectory*.
+
+.. code-block:: python
+  :caption: :file:`run_command.py`
+
+    component: MTDomeTrajectory
+    cmd: setFollowingMode
+    parameters:
+        enable: false
+..
+
+2. To clear the subsystem ``FAULT``, use the MTDome **exitFault** configuration command via the :file:`run_command.py` SAL script. 
+In most cases, running the command once will clear the ``FAULT``, but you may need to repeat the process a few times.
+
+.. code-block:: python
+  :caption: :file:`run_command.py`
+
+    component: MTDome
+    cmd: exitFault
+..
+
+3. After executing the command, verify its success by checking the ACS status in the `MTDome Status`_ dashboard. 
+
+.. Note::
+    When the **exitFault** command is sent, the CSC executes the *resetDrivesAz* command before **exitFault**  command.
+
+.. code-block:: python  
```

Comment:
I suggest use as code-block the text mode instead of python. The script is a SAL script is reality.
.. code-block:: python -->> .. code-block:: text